### DISCUSSION TITLE
Backing Up Settings and Passwords

### DISCUSSION THREAD

**Discussion by erayerdin** [Eloston/ungoogled-chromium#2402] [Q&A] [answered]
How do you back up settings and passwords in Ungoogled Chromium? I'd like to at least have it tarred and upload to my cloud.

**Comment by PF4Public** [comment, marked as answer]
One could backup the whole profile. Passwords are normally not recognized if profile was copied, but there is password import/export functionality in password manager:
![image](https://github.com/ungoogled-software/ungoogled-chromium/assets/10319700/78c19014-9148-4384-adf1-e1829fc84358)
